components/WorkoutForm.js

```diff
@@ -0,0 +1,268 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { exercises } from "@/lib/exercises";
+import findExerciseById from "@/utils/findExerciseById";
+
+// Styled Components
+const Header = styled.header`
+  text-align: center;
+  margin-bottom: 1rem;
+`;
+
+const WrapperFilter = styled.div`
+  width: 100%;
+  height: 100%;
+  position: fixed;
+  top: 0;
+  z-index: 1;
+  background: rgba(0, 0, 0, 0.6);
+  padding: 1rem 3rem;
```

Comment:
I wouldn't recommend using rem for paddings and margins because rem is relative to the font-size set in the broweser settings. If somebody sets the font-size to twice or three times the default amount, the padding/margin would become really huge as well.